extmod/modlwip.c

```diff
@@ -1711,6 +1711,10 @@ STATIC void lwip_getaddrinfo_cb(const char *name, const ip_addr_t *ipaddr, void
     }
 }
 
+#if LWIP_VERSION_MAJOR >= 2
+extern int mp_mod_network_prefered_dns_ip;
```

Comment:
I misunderstood this initially to be an ip address of a DNS server. Please rename to `mp_mod_network_prefer_dns_use_ip_version`